Lib/test/test_generated_cases.py

```diff
@@ -458,7 +459,7 @@ def test_predictions(self):
             INSTRUCTION_STATS(OP3);
             static_assert(INLINE_CACHE_ENTRIES_OP1 == 0, "incorrect cache size");
             _PyStackRef res;
-            DEOPT_IF(xxx, OP1);
+            GO_TO_INSTRUCTION_IF(xxx, OP1, INLINE_CACHE_ENTRIES_OP1);
```

Comment:
Rather than replace one magic macro with another, could we emit the actual C code?
We should know which interpreter we are generating code for at this point.

Reply:
Replaced with actual C code.